Owlcat/FitsTool.py

```diff
@@ -629,8 +630,16 @@ def compute_in_out_slice(N, N0, R, R0):
 
     if options.zoom:
         z = options.zoom
+
+        if ',' in z:
+            zx = int(z.split(',')[0])
+            zy = int(z.split(',')[1])
```

Comment:
It truly doesn't matter, but my OCD suggests the one-liner ``zx, zy = map(int, z.split(','))``, just because I always feel sorry asking the computer to do the same work twice! My one small blow against climate change.